src/main/java/com/smalaca/messagesender/controller/UserController.java

```diff
@@ -4,31 +4,44 @@
 import com.smalaca.messagesender.service.UserCrud;
 import com.smalaca.messagesender.service.UserDto;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.ModelAttribute;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
 
+@RequestMapping("/user")
 @RestController
 public class UserController {
+    public static final String FORBIDDEN_LOGIN = "";
 
     @Autowired
     private UserCrud userCrud;
 
-    @RequestMapping("/user/create")
-    public String createUser(@ModelAttribute UserDto userDto) {
+    @RequestMapping("/create")
+    public ResponseEntity<String> createUser(@ModelAttribute UserDto userDto) {
         if (userCrud.createUser(userDto))
-            return "User Created";
+            return new ResponseEntity<>("User Created", HttpStatus.OK);
         else
-            return "User Creation Failed!";
+            return new ResponseEntity<>("User Creation Failed!", HttpStatus.OK);
     }
 
-    @RequestMapping("/user/block")
-    public String blockUser(@RequestParam String login) {
+    @RequestMapping("/block")
+    public ResponseEntity<String> blockUser(@RequestParam(value = "login", defaultValue = FORBIDDEN_LOGIN) String login) {
```

Comment:
nie ma czegoś takiego jak ```FORBIDDEN_LOGIN``` 😄 
1) Albo obsłużcie sytuację, gdy loginu nie ma.
2) Albo Wykorzystajcie obiekt (UserDto???), które jedym z parametrów może, być login. Jeżeli go nie będzie to po prostu pole będzie null.